oxd-server/src/main/java/org/gluu/oxd/server/Utils.java

```diff
@@ -116,4 +118,29 @@ public static Date addTimeToDate(Date date, int timeValue, int timeFormat) {
         cal.add(timeFormat, timeValue);
         return cal.getTime();
     }
+
+    public static Properties loadPropertiesFromFile(String filename, Properties props) {
+
+        FileInputStream fileistream = null;
+        try {
```

Comment:
Just try-with-resource, no need to close it manually : 
```
try(FileInputStream fileistream = new FileInputStream(filename)) {
    ....
}
```

Reply:
Done